Autopilot/Src/MPXV7002DP.cpp

```diff
@@ -0,0 +1,92 @@
+#include "airspeed.hpp"
```

Comment:
I really like how you named the file. We may have to adopt this for all sensors.